examples/02_model_collaborative_filtering/standard_vae_deep_dive_2.ipynb

```diff
@@ -0,0 +1,2268 @@
+{
```

Comment:
<p>If you guys are going to leave these two files, I guess you want to change the name and remove <code>_2</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>